Adds QA updates as agreed upon for tag: Futuristic

kalapi opened the pull request:
Updates tag data for 'Futuristic' with decisions made in [this sheet](https://docs.google.com/spreadsheets/d/15ivqgq3HHAfFqhHgbm7PfxJw71Dkxxa9o_ycZ_Wde_c/edit?gid=185429034#gid=185429034).

cc: @evanwadams @m4rc1e

evanwadams commented:
I'd like to keep this set of PRs in sync with the spreadsheet.

Most likely, your ad hoc changes are good changes, but let's do them in a separate pass.

kalapi commented:
@evanwadams I fixed the conflicts in this branch, it was 4 families we removed from Business:

- Graduate
- Holtwood One SC
- Stint Ultra Condensed
- Vast Shadow

I've accepted the incoming changes, i.e. agree to remove.
